core/src/test/java/org/frankframework/http/rest/ApiListenerServletTest.java

```diff
@@ -913,6 +926,33 @@ public void apiListenerWithRepeatableMessageAndGloballyDisabled() throws Excepti
 		assertTrue(result.containsHeader("pragma"));
 	}
 
+	@Test
+	public void apiListenerWithHeadMethodCall() throws Exception {
+		// Arrange
+		String uri = "/apiListenerWithHeadMethodCall";
+		Message repeatableMessage = Message.asMessage(new Message("{\"tralalalallala\":true}").asByteArray());
```

Comment:
After applying other comments to this test, I would like a copy of this test where this `repeatableMessage` is constructed with an empty string, but the size set to something like 20 by setting the corresponding key in the MessageContext. Then the content-length returned by HEAD should also be 20.